src/services/RoutingService.js

```diff
@@ -80,6 +83,67 @@ import { bvvRouteStatsProvider } from './provider/routeStats.provider';
  * @returns {Promise<Array<module:domain/routing~RoutingCategory>>} available categories
  */
 
+/**
+ * Returns all available surface type styles
+ * @function
+ * @name module:services/RoutingService~RoutingService#getSurfaceTypeStyles
+ * @returns {Array<module:domain/routing~ChartItemStyle>} surfaceTypeStyles
+ */
+
+/**
+ * Returns all available road type styles
+ * @function
+ * @name module:services/RoutingService~RoutingService#getRoadTypeStyles
+ * @returns {Array<module:domain/routing~ChartItemStyle>} roadTypeStyles
+ */
+
+/**
+ * A function that returns a list of available ChartItems to display statistical
+ * data (road type, surface type) for routing
+ * @function
+ * @typedef {Function} chartItemStylesProvider
+ * @returns {Map<string,module:domain/routing~ChartItemStyle>} available chartItems
+ */
+
+/**
+ * A default {@link module:domain/routing~ChartItemStyle} object for chart items of road-classes
+ * if the {@link module:services/RoutingService~chartItemStylesProvider} cannot
+ * provide a specific style
+ */
+export const CHART_ITEM_ROAD_STYLE_UNKNOWN = {
+	id: 0,
+	color: 'transparent',
+	image: 'repeating-linear-gradient(45deg,#eee 0px,#eee 7px, #999 8px, #999 10px, #eee 11px)',
+	label: 'Unknown'
+};
+
+/**
+ * A default {@link module:domain/routing~ChartItemStyle} object for chart items of surface-classes
+ * if the {@link module:services/RoutingService~chartItemStylesProvider} cannot
+ * provide a specific style
+ */
+export const CHART_ITEM_SURFACE_STYLE_UNKNOWN = {
+	id: 0,
+	color: 'transparent',
+	image: 'repeating-linear-gradient(45deg,gray 25%, transparent 25%,transparent 50%, gray 50%, gray 55%, transparent 55%, transparent)',
+	label: 'Unknown'
+};
+
+/**
+ * A function that maps and reduces {@link module:domain/routing~RouteDetailTypeAttribute}
+ * with OSM road type names to the name of defined {@link module:domain/routing~ChartItemStyle}
```

Comment:
Isn't it a method of the `RoutingService`?
`@param routeDetailTypeAttributes` is missing

Reply:
Yes, i change that....but the provider should also reference this ChartItemStyle to provide (override) a i18n-depending label property